src/drf_yasg/utils.py

```diff
@@ -1,6 +1,7 @@
 import inspect
 import logging
 import textwrap
+import zoneinfo
```

Comment:
`zoneinfo` was introduced from Python 3.9, shall we consider checking Python version when importing it?

Reply:
Oh, you're right, thanks. I fixed it.